README.md

```diff
@@ -9,6 +9,8 @@ Windows: `venv\Scripts\activate`<br>
 macOS and Linux: `source venv/bin/activate`
 
 - Next, run `pip install -e .` to install the package with its dependencies.
+- Open a terminal tab and run the image `docker run --rm -p 8082:8070 elifesciences/sciencebeam-parser` and keep it running for the rest of the testing period
```

Comment:
These instructions mention macOS... the scibeam parser docker image doesn't work on the M1 architecture so it might be better to warn about that and at least remove that os from the README.

Can you remove the port mapping to 8082. Let's just keep it at 8070 and if there's any need we can make it configurable at some point.

Reply:
We have updated the wordings to include a note for Apple silicon chip users.